R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -37,16 +39,23 @@
 #'                                  package = "biomformat")
 #'   se <- loadFromBiom(rich_dense_file, removeTaxaPrefixes = TRUE,
 #'                     rankFromPrefix = TRUE)
-#'   # or if the file contains some character artifacts at the taxonomy data,
-#'   # the character(s) to be removed could be provided (also as regular expression)
-#'   #se <- loadFromBiom(rich_dense_file, removeTaxaPrefixes = TRUE,
-#'   #                  rankFromPrefix = TRUE, cleanTaxaPattern="\"")
+#'                     
 #'   # load from object
 #'   x1 <- biomformat::read_biom(rich_dense_file)
 #'   se <- makeTreeSEFromBiom(x1)
 #'   # Convert SE to TreeSE
 #'   tse <- as(se, "TreeSummarizedExperiment")
 #'   tse
+#'   
+#'   # Cleaning artifacts from Taxonomy data
+#'   biom_object <- biomformat::read_biom(
+#'       system.file("extdata/testdata/Aggregated_humanization2.biom",
+#'                   package="mia"))
```

Comment:
might be more clear to first specify this as file name 

`f <- system.file(...)`

and then on separate line run `read_biom(f)`

is users want to copypaste the example they can just change the file name